demisto_sdk/commands/content_graph/tests/SC_graph_validation_test.py

```diff
@@ -4,10 +4,14 @@
 )
```

Comment:
I don't think validation tests should seat here, they should be in the validate folder.
@YuvHayun WDYT?

Reply:
i moved it to the validate folder and its works